app/views/shared/_form_photocopy.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="form-group">
+  <% if entity.present? and entity.send(attribute).present? %>
```

Comment:
Pretty sure this was in the original code from RJ, but that totally makes sense.